doc/Clockwork installation.md

```diff
@@ -0,0 +1,5 @@
+Find concerto-clock.service (currently in concerto/script)\
```

Comment:
Like the other file, I suggest adding a `# header` or title to each markdown file before jumping straight into the content.